project.clj

```diff
@@ -62,7 +62,7 @@
                                   [puppetlabs/trapperkeeper ~tk-version :classifier "test"]
                                   [org.clojure/tools.namespace "0.2.4"]
                                   [org.clojure/java.jmx "0.2.0"]
-                                  [spyscope "0.1.4"]
+                                  [spyscope "0.1.5" :exclusions [clj-time]]
```

Comment:
Fair enough.  I'll go with the approach of just adding a hard clj-time dependency on 0.5.1 and just staying with spyscope 0.1.4 for now.  I'll leave the bump of the clojure dependency to 1.6.0 since it seems like we're going that way for other projects right now anyway.